src/Callback.php

```diff
@@ -81,18 +65,40 @@ public function init(): void
         $this->_init();
 
         if (!$this->app) {
-            throw new Exception('Call-back must be part of a RenderTree');
+            throw new Exception('Callback must be part of a render tree');
         }
 
-        if (!$this->urlTrigger) {
-            $this->urlTrigger = $this->name;
-        }
+        $this->setUrlTrigger($this->urlTrigger);
+    }
 
-        if ($this->postTrigger === true) {
-            $this->postTrigger = $this->name;
-        }
+    public function setUrlTrigger(string $trigger = null)
+    {
+        $this->urlTrigger = $trigger ?? $this->name;
+    }
 
-        $this->appSticky ? $this->app->stickyGet($this->urlTrigger) : $this->owner->stickyGet($this->urlTrigger);
+    public function getUrlTrigger(): string
+    {
+        return $this->urlTrigger;
+    }
+
+    private function callWithAppStickyTrigger(\Closure $fx)
```

Comment:
Why do we need this? All was working perfectly fine before...

Reply:
not, it was not - the described reload issue on Discord is not resolved, I belive, it never will with app/global sticky args

This should be the same as before, but only the app/global sticky args are set when the callback is executing - once it is done, it is no longer app/global sticky - we need to converge to a solution without app/global sticky at all

Reply:
I do not see any issue using your branch describe on Discord. All was working fine for me locally. I can even click 'Add Country" three time and Add will still work using the third modal.

Reply:
https://discord.com/channels/657649422280425523/664534590160896006/737766184044199947

how to reproduce https://discord.com/channels/657649422280425523/664534590160896006/738128513412104303